### ISSUE TITLE
Quits Chrome instead if you try to ⌘Q a Chrome App

### ISSUE THREAD

**Issue by 00dani** [dteoh/SlowQuitApps#7]
With SlowQuitApps enabled, holding ⌘Q with Chrome focused quits it, regardless of the Warn Before Quitting setting in Chrome (it's ignored). This is not a problem, since SlowQuitApps gives you its own warning anyway.

However if you hold ⌘Q with a Chrome _app_ focused, such as [Authy](https://chrome.google.com/webstore/detail/authy/gaedmjdfmmahhbjefcbgaolhhanlaolb?hl=en) or [Wunderlist](https://chrome.google.com/webstore/detail/wunderlist-for-chrome/ojcflmmmcfpacggndoaaflkmcoblhnbh?hl=en), then the Chrome app stays running and the underlying Chrome browser quits instead. If you then attempt to close the app with the mouse, Chrome _crashes_:

![screen shot 2016-10-02 at 14 46 37](https://cloud.githubusercontent.com/assets/610876/19018494/4afaed2a-88af-11e6-8e21-f24858436aa1.png)

(Here're the full details from that crash: [error.txt](https://github.com/dteoh/SlowQuitApps/files/504769/error.txt))

With SlowQuitApps disabled, hitting ⌘Q in a Chrome app exits the app immediately and does not quit Chrome. No crashes.

**Comment by dteoh** [comment]
My preliminary findings are that Chrome Apps are not standalone apps (meaning that they are detached from the parent process). I noticed that:

* when I focus a Chrome app, there is some menu bar trickery going on. Notice that the App's name text is not bold. I believe the App is just a sub-window, and when the sub-window is active, the "Chrome" menu is hidden/disabled.
* when I quit Chrome while an App is running, Chrome closes its windows, but doesn't quit until the App quits.

I did not experience a crash when I tested with the Authy app. It is also strange that Chrome says that it quit unexpectedly, because SQA does not force quit an application.

This will probably take a while to resolve. At this stage, I think the solution involves getting AppleScript to trigger the quit menu item.
